RiotSwiftUI/Modules/UserSessions/UserSessionDetails/View/UserSessionDetailsItem.swift

```diff
@@ -17,15 +17,8 @@
 import SwiftUI
 
 struct UserSessionDetailsItem: View {
-    
-    // MARK: - Properties
-    
-    // MARK: Private
```

Comment:
What is the rule about the auto-generated template comments, do we keep or remove them? I am asking this because after removing them in one of my PRs, I was asked to put them back.

Reply:
Yeah, I'm coming at this with fresh eyes after using the templates in ElementX. I think we should remove them and I'll go update the templates as well. I really don't believe they bring any value.

Reply:
On a related note, we really need to add swiftformat to at least the RiotSwiftUI target, it will massively help us clean up the code.

Reply:
haha, I read this comment only after I posted a comment about SwiftFormat 😄